.github/ISSUE_TEMPLATE.md

```diff
@@ -3,12 +3,13 @@
 > what info we expect.  
 > Please remove this line and all above before submitting.
 
-> Before you start, are you using the CocoaPods beta release?
+> Before you start, are you using the CocoaPods release?
 > a lot has changed with Xcode 8 that is not backwards compatible.
 >
-> Xcode 8 Status: Only works with CocoaPods 1.1.0
-> Swift 3: You need to add a `.swift-version` file, see the CHANGELOG
-> China: Use https://gems.ruby-china.org/ instead of https://gems.ruby-china.org/ for the RC see #5910
```

Comment:
I would also add something along the lines of `Using CocoaPods <= 0.39? See [link to blog post]`